src/components/UserAccount/index.js

```diff
@@ -33,7 +40,14 @@ function UserAccount({ ...props }) {
       </Grid>
       <Divider />
       <div className={classes.buttons}>
-        <Button variant="contained" color="primary" className={classes.button}>
+        <Button
+          component={Link}
+          href="/account/update"
```

Comment:
```suggestion
          href="/account/settings/update"
```
To show we're updating the account settings page